src/main/java/org/peergos/HostBuilder.java

```diff
@@ -92,8 +92,7 @@ public HostBuilder generateIdentity() {
         return setPrivKey(Ed25519Kt.generateEd25519KeyPair().getFirst());
     }
 
-    public HostBuilder setIdentity(String base64PrivKey) {
-        byte[] privKey = Base64.decodeBase64(base64PrivKey);
+    public HostBuilder setIdentity(byte[] privKey) {
```

Comment:
Can we name this something like setEd25519Identity?

Reply:
We need to support existing RSA identity.

Reply:
done